scripts/modules.rollup.config.js

```diff
@@ -9,8 +9,8 @@ import babel from 'rollup-plugin-babel';
 import json from 'rollup-plugin-json';
 
 export default {
-  moduleName: 'm3u8-parser',
-  entry: 'src/index.js',
+  name: 'm3u8_parser',
```

Comment:
This is something I'm not sure on - `rollup` will complain about the hyphen, but I can't tell if this would then require a change to the import statements for things that require `m3u8-parser`?

Reply:
It should be fine, but I think we should go with camelcase (not a big deal, just preference).